components/badge/badge.css

```diff
@@ -0,0 +1,70 @@
+@import url("https://fonts.googleapis.com/css2?family=Source+Sans+Pro:ital@1&family=Ubuntu:wght@300&display=swap");
+body {
+  font-family: "Source Sans Pro", sans-serif;
+  font-family: "Ubuntu", sans-serif;
+}
+
+* {
+  --primary-color: red;
+  --secondary-color: white;
+  --tertiary-color: #2563eb;
+}
+
+body {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.badge {
+  position: relative;
+  font-size: 13px;
+  font-weight: bold;
+  background-color: var(--primary-color);
+  border-radius: 32px;
+  width: max-content;
+  color: var(--secondary-color);
+}
+
+.text {
+  margin: 32px;
+  font-weight: bold;
+}
+
+.badge-on-text {
+  top: -13px;
+  left: -5px;
+  padding: 4px 8px;
+}
+
+.large-badge-on-text {
+  top: -13px;
+  left: -5px;
+  padding: 8px 8px;
+}
+
+.button1 {
+  padding: 10px 13px;
+  border-radius: 8px;
+  background-color: var(--tertiary-color);
+  color: var(--secondary-color);
+  border: 0px;
+  font-size: 16px;
+}
+
+.badge-on-button {
+  top: -62px;
+  right: 14px;
+  padding: 3px 8px;
+}
+
+.fas {
```

Comment:
dont-use fas class for the styling because maybe the user want to use some other icon library and then it will not work as it became very specific for font awesome only. you can add something like a badge-icon as the class and do the same.